src/viking.js

```diff
@@ -1,11 +1,162 @@
 // Soldier
-class Soldier {}
+class Soldier {
+
+    constructor (health,strength) {
+    this.health = health;
+    this.strength = strength
+}
+//metodos soldier
+attack() {
+     
+    return this.strength
+};
+
+receiveDamage(damage) {
+
+    this.health = this.health - damage
+}
+}
 
 // Viking
-class Viking {}
+class Viking extends Soldier {
+
+    constructor (name,health , strength) {
+      super (name,health , strength)
+
+    this.name = name
+    this.health = health
+    this.strength = strength }
```

Comment:
Soy un pijo lo se pero estos detalles marcan la diferencia!
```suggestion
    constructor (name, health, strength) {
      super (name,health , strength)
      this.name = name
      this.health = health
      this.strength = strength
    }
```